### ISSUE TITLE
[Settings - Line Height] : Screen reader is not announcing the sub-text present under the 'Line Height' button.

### ISSUE THREAD

**Issue by Saiteja341** [microsoft/terminal#15157]
### Windows Terminal version

1.18.902.0

### Windows build number

25324.1011

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 25324.1011)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

Repro Steps:

1.Open Windows Terminal.
2.Open Setting page using 'Ctrl + ,' keys.
3.Now navigate to the 'Advanced' button present under 'Default Profile'.
4. Now navigte to the 'Line Height' and Observe the issue.

User Experience:
People with visual impairments who rely on the screen readers are unable to know the button's description.

Attachments:
[Screen reader is not announcing the sub-text present under the 'Line Height' button..zip](https://github.com/microsoft/terminal/files/11197360/Screen.reader.is.not.announcing.the.sub-text.present.under.the.Line.Height.button.zip)

### Expected Behavior

Screen reader should announce the sub-text present under the 'Line Height' button.

### Actual Behavior

Screen reader is not announcing the sub-text present under the 'Line Height' button.
